lambdas/src/apis/collections/off-chain/OffChainWearablesManager.ts

```diff
@@ -10,17 +10,21 @@ import baseAvatars from './base-avatars'
  * then look at 'baseAvatars' at the end of the file.
  */
 export class OffChainWearablesManager {
-  private definitions: IFuture<LocalOffChainWearables>
+  private readonly definitions: TimeRefreshedDataHolder<LocalOffChainWearables>
 
   constructor(
-    private readonly client: SmartContentClient,
-    private readonly collections: OffChainCollections = DEFAULT_COLLECTIONS
-  ) {}
+    client: SmartContentClient,
+    collections: OffChainCollections = DEFAULT_COLLECTIONS,
+    refreshTime: string = '1d'
```

Comment:
Can we make the refresh time configurable?

Reply:
Done!